_config.yml

```diff
@@ -28,3 +28,4 @@ sass:
 timezone: Asia/Tokyo
 title: Example title
 url: http://example.com
+google_analytics:
```

Comment:
For consistency, the keys in YAML files are sorted in alphabetical order. Can you put `google_analytics:` before `hashtag: ...` ?